GRADLE_STRUCTURE.md

```diff
@@ -0,0 +1,16 @@
+# Gradle Structure
```

Comment:
Having seen the details here, I think a wiki page referenced from https://github.com/flutter/flutter/wiki/Understanding-Packages-tests#external-build-dependencies and maybe https://github.com/flutter/flutter/wiki/Plugins-and-Packages-repository-structure would probably be better than an md file.

Reply:
ok in that case consider the md files content for review and I will remove and update the wiki before merging.